static/app/views/settings/account/apiApplications/details.tsx

```diff
@@ -10,131 +9,160 @@ import Confirm from 'sentry/components/confirm';
+        saveOnBlur
+        allowUndo
+        initialData={app}
+        onSubmitError={() => addErrorMessage('Unable to save change')}
+      >
+        <JsonForm forms={apiApplication} />
+
+        <Panel>
+          <PanelHeader>{t('Credentials')}</PanelHeader>
+
+          <PanelBody>
+            <FormField name="clientID" label="Client ID">
+              {({value}) => (
+                <TextCopyInput>
+                  {getDynamicText({value, fixed: 'CI_CLIENT_ID'})}
+                </TextCopyInput>
+              )}
+            </FormField>
+
+            <FormField
+              name="clientSecret"
+              label="Client Secret"
+              help={t(`Your secret is only available briefly after application creation. Make
                   sure to save this value!`)}
-              >
-                {({value}) =>
-                  value ? (
-                    <TextCopyInput>
-                      {getDynamicText({value, fixed: 'CI_CLIENT_SECRET'})}
-                    </TextCopyInput>
-                  ) : (
-                    <ClientSecret>
-                      <HiddenSecret>{t('hidden')}</HiddenSecret>
-                      <Confirm
-                        onConfirm={this.rotateClientSecret}
-                        message={t(
-                          'Are you sure you want to rotate the client secret? The current one will not be usable anymore, and this cannot be undone.'
-                        )}
-                      >
-                        <Button size="xs" priority="danger">
-                          Rotate client secret
-                        </Button>
-                      </Confirm>
-                    </ClientSecret>
-                  )
-                }
-              </FormField>
-
-              <FormField name="" label="Authorization URL">
-                {() => <TextCopyInput>{`${urlPrefix}/oauth/authorize/`}</TextCopyInput>}
-              </FormField>
-
-              <FormField name="" label="Token URL">
-                {() => <TextCopyInput>{`${urlPrefix}/oauth/token/`}</TextCopyInput>}
-              </FormField>
-            </PanelBody>
-          </Panel>
-        </Form>
-      </div>
-    );
-  }
+            >
+              {({value}) =>
+                value ? (
+                  <TextCopyInput>
+                    {getDynamicText({value, fixed: 'CI_CLIENT_SECRET'})}
+                  </TextCopyInput>
+                ) : (
+                  <ClientSecret>
+                    <HiddenSecret>{t('hidden')}</HiddenSecret>
+                    <Confirm
+                      onConfirm={rotateClientSecret}
+                      message={t(
+                        'Are you sure you want to rotate the client secret? The current one will not be usable anymore, and this cannot be undone.'
+                      )}
+                    >
+                      <Button size="xs" priority="danger">
+                        Rotate client secret
```

Comment:
```suggestion
                       {t("Rotate client secret")}
```

Reply:
I noticed that the translations are missing in a couple of places here but it can be done in a separate PR